tests/network-tests/src/misc/postMigrationAssertionsFlow.ts

```diff
@@ -0,0 +1,50 @@
+import { assert } from 'chai'
+import { FlowProps } from '../Flow'
+import { extendDebug } from '../Debugger'
+
+export default async function postMigrationAssertions({ api }: FlowProps): Promise<void> {
+    const debug = extendDebug('flow:postMigrationAssertions')
+    debug('Started')
+
+    debug('Ensure migration is done')
+
+    const channel_migration = await api.query.content.channelMigration();
+    const video_migration = await api.query.content.videoMigration();
+
+    assert(channel_migration.current_id.toNumber() == channel_migration.final_id.toNumber())
+    assert(video_migration.current_id.toNumber() == video_migration.final_id.toNumber())
```

Comment:
When I ran locally I got the channel assertion passing but the video assertion failing.

Reply:
```
yarn run v1.22.4
$ node -r ts-node/register --unhandled-rejections=strict src/scenarios/post-migration.ts
AssertionError: expected 7600 to equal 8192
```
When I ran with assert.equal(..)

Reply:
I guess it's due to the sleep 600 line (underestimation). I fixed it by adding a while loop